package.json

```diff
@@ -82,13 +82,15 @@
 		"jsdom": "^3.1.2",
 		"mocha": "^2.2.4",
 		"jscs": "~1.6.2",
-		"jshint": "^2.6.0"
+		"jshint": "^2.6.0",
+		"nan": "^1.8.4"
 	},
 	"engines": {
-		"node": ">=0.10"
+		"node": ">=0.12"
```

Comment:
The V8 version is changed since 0.10 and its interface is changed dramatically for native modules. I don't think it's necessary to support this old version because now we have even iojs 2.x branch as a stable release. I think Node.js 0.x versions will be dropped in support soon after Node foundation was created.